_docker/drupal/drupal-filesystem/web/modules/custom/rhd_purpose_attributes/src/Controller/PurposeAttributesPageController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace Drupal\rhd_purpose_attributes\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+use Drupal\Core\Link;
+use Drupal\core\Url;
+
+/**
+ * Class PurposeAttributesPageController.
+ *
+ * @package Drupal\rhd_purpose_attributes\Controller
+ */
+class PurposeAttributesPageController extends ControllerBase {
+
+  /**
+   * Returns a purpose_attributes_report_block block.
+   *
+   * @return array
+   *   Returns a block render array.
+   */
+  public function renderBlockContent() {
+    $plugin_block = \Drupal::service('plugin.manager.block')
```

Comment:
Can we inject the `plugin.manager.block` service in the class and use the attribute here?